app/views/home/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<section class="bold main-text">SnapScan</section>
+
+<section class="login-btns container">
+    <%= button_to "LOG IN" , new_user_session_path, class: "login-btn" %>
+    <%= link_to "SIGN UP" , new_user_registration_path, class: "btn btn-primary" %>
+</section>
```

Comment:
- [ ] I will suggest adding some space between the two buttons to much the design.